client/src/Components/SearchPageComponents/CardResultFilter/Card.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+import {
+  Box,
+  Button,
+  CardContent,
+  Chip,
+  Divider,
+  Rating,
+  Typography,
+} from "@mui/material";
+import { Link } from "react-router-dom";
+
+export default function Card({ filter }) {
+  const { id, name, image, location, description, rating, features } = filter;
+
+  return (
+    <div className="card__filter">
+      <div className="image__card">
+        <img alt="complex" src={image} />
+      </div>
+      <div className="body__card">
+        <Typography sx={{ m: 1, mb: 0 }} variant="h4">
+          {name}
+        </Typography>
+        <Typography sx={{ mx: 1 }} variant="body3">
+          {location}
+        </Typography>
+        <CardContent>
+          <Typography variant="body2" color="text.secondary">
+            {description.slice(0, 100)}...
+          </Typography>
+        </CardContent>
+        <Divider variant="middle" />
+        <Rating value={rating} readOnly sx={{ m: 1 }} />
+        <Box sx={{ m: 1 }}>
+          <Typography gutterBottom variant="body1">
+            المزايا
+          </Typography>
+          <Box sx={{ display: "flex", flexWrap: "wrap", gap: "0.5rem" }}>
+            {features.map((feature) => (
+              <Chip key={feature} variant="outlined" label={feature} />
+            ))}
+          </Box>
+        </Box>
+
+        <Box
+          sx={{ m: 1, mt: "auto", display: "flex", justifyContent: "flex-end" }}
+        >
+          <Link to={`/gyms/profile/${id}`}>
+            <Button variant="contained">احجز موعد</Button>
```

Comment:
No need to have a button here,
Just create a div or span and customize style to fit your design